sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -69,15 +69,18 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
         if (this.model.get('saved')) {
             this.model.save({saved: false},
                 {success: this.savedToast});
-
-            if (this.model.changedAttributes()) {
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                        "Feed Title": this.model.feed.get('title') });
-            }
-
         } else {
             this.model.save({saved: true},
                 {success: this.savedToast});
+
+            if (this.model.changedAttributes()) {
```

Comment:
Oh yeah. You're right. A user at the moment can click upvote a bunch of times and retrigger the signal. Maybe we should fix the Backbone event to only register the click if it's not active? Then we would be able to remove this.